pkg/cli/image/imagesource/file_test.go

```diff
@@ -0,0 +1,20 @@
+package imagesource
+
+import (
+	"path/filepath"
+	"runtime"
+	"testing"
+)
+
+func TestGenerateDigestPath(t *testing.T) {
+	path := generateDigestPath("sha256:123456", "mnt", "mirror")
+	desiredPath := filepath.Join("mnt", "mirror", "sha256:123456")
+
+	if runtime.GOOS == "windows" {
+		desiredPath = filepath.Join("mnt", "mirror", "sha256-123456")
```

Comment:
I'd suggest not using `filepath.Join` but rather use a hardcoded string for win and non-win.